### PR TITLE
Enhancement: Configure no_extra_consecutive_blank_lines fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] configures the `no_extra_consecutive_blank_lines` fixer 
* [x] runs `make cs`

### CODE REVIEW

File: classes/Domain/Talk/TalkFilter.php [hunk 1]
```diff
@@ -71,25 +71,18 @@ public function getFilteredTalks(int $adminUserId, $filter = null)
...
-
             case 'plusone':
                 return $this->talk->ratedPlusOneBy($adminUserId);
-
             case 'viewed':
                 return $this->talk->viewedBy($adminUserId);
-
             case 'favorited':
                 return $this->talk->favoritedBy($adminUserId);
-
```
Comment (derrabus) [inline classes/Domain/Talk/TalkFilter.php]: I kinda like blank lines before `case`/`default`, I must say. 😕

Comment (localheinz) [reply classes/Domain/Talk/TalkFilter.php]: I don't mind removing the configuration for `case` statements, as I personally never use `switch` statements.

Comment (localheinz) [reply classes/Domain/Talk/TalkFilter.php]: This case is actually handled by configuring the `return` token: it removes extra consecutive blank lines after a `return` statement.

**Comment by localheinz** [comment]
Thank you, @BackEndTea!

**Pull request merged**